spec/support/shared_examples/good_responses.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.shared_examples 'good responses' do
```

Comment:
``
```suggestion
RSpec.shared_examples 'successful response' do
```

donde se usa?

Reply:
Hice esta corrección en otro PR mergeado a master, me lo había marcado Gonza.